packages/teleport/src/services/websession/storeWebSession.ts

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import history from 'teleport/services/history';
+
+import {
+  KeysEnum,
+  EventKeys,
+  BroadcastChannelMessage,
+  BearerToken,
+} from './types';
+
+const isTest = process.env.NODE_ENV === 'test';
```

Comment:
Generally you want to try and void adding test specific code, especially toggling on a 'test environment' as any resulting tests will not be testing the true execution of the code.

I also don't see a test file for this?

Reply:
Agreed - we're introducing side effects where things run differently in our test environment vs web app